extended/jams01ex.txt

```diff
@@ -13,10 +13,6 @@
           "label": { "value": 1 },
           "time": { "value": 1.196 }
         },
```

Comment:
I found that the two big json files were really hard to read, so I increased the font size very slightly (re. your comment below), which pushed them over the edge on the page, so I removed these lines. Any protests?